src/redprl/refiner.fun

```diff
@@ -383,11 +382,6 @@ struct
      | UPDATE of hyp * catjdg
      | INSERT of hyp * catjdg
 
-    val diffToString =
-      fn DELETE x => "DELETE " ^ Sym.toString x
-       | UPDATE (x,_) => "UPDATE " ^ Sym.toString x
-       | INSERT (x,_) => "INSERT " ^ Sym.toString x
-
```

Comment:
I would keep it; it's kind of annoying to have to rewrite the same code again and again when I need to debug something.

Ultimately I think that this falls into the same category as our other discussion thread: not every bit of unused code is a bug. Sometimes it is the right thing to have unused code or unused variables.